EFSAdvent/data/actorinfo/DOOR.txt

```diff
@@ -1,4 +1,4 @@
-DOOR (door)
+DOOR (door) Has GBA sprite
 A doorway.
```

Comment:
Most tile based actors do not have sprits, perhaps it would be better to consider this?